demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -25,23 +24,26 @@ def test_integration_secrets_incident_field_positive(mocker):
     - Ensure secrets validation passes.
     - Ensure success secrets validation message is printed.
     """
+    # Mocking the git functionality (Else it'll raise an error)
+    mock_git(mocker)
+    # Change working dir to repo
+    os.chdir(integration.repo_path)
     mocker.patch(
         "demisto_sdk.__main__.SecretsValidator.get_all_diff_text_files",
         return_value=[
-            join(DEMISTO_SDK_PATH,
-                 "tests/test_files/content_repo_example/Packs/FeedAzure/IncidentFields/incidentfield-city.json"
-                 )
+            integration.yml_path
         ]
     )
+    integration.write_yml({'valid yml file': 'really'})
     runner = CliRunner(mix_stderr=False)
-    result = runner.invoke(main, [SECRETS_CMD, '-wl', SECRETS_WHITELIST])
+    result = runner.invoke(main, [SECRETS_CMD, '-wl', integration.global_secrets.path])
```

Comment:
it's pointing to the global secrets @ the repo.
Maybe should pass the repo itself to every step in the sdk. (integration.repo, pack.rep)

Reply:
lets just pass a repo. instead of integration or any other content item.
The test will always get a repo, and will start from that point.
```python
def test_validate(repo):
      pack = repo.create_pack(name='foo_pack')
      integration = pack.create_integration(name='goo_integration')
```